### DISCUSSION TITLE
Intellisense does not show the complete multiline description of function/method parameters

### DISCUSSION THREAD

**Discussion by diogo-rossi** [microsoft/pylance-release#2648] [Enhancement]
## Environment data

-   VS Code version: 1.60.0
-   Extension version: 2021.9.1218897484
-   OS and version: Windows_NT x64 10.0.19043
-   Python version:  3.8.8, Anaconda
-   Type of virtual environment used: conda
-   Relevant/affected Python packages and their versions: none
-   Relevant/affected Python-related VS Code extensions and their versions: Pylance 2021.9.1
-   Value of the `python.languageServer` setting: Default

## Expected behaviour

Intellisense should show descriptions of function/method parameters of the Docstring as you type then, like the example below.

![gif000](https://user-images.githubusercontent.com/57204023/132867089-a8cd91aa-4eeb-4b13-8682-6bb665d99f92.gif)

## Actual behaviour

When the descriptions of function/method parameters **takes more than one line in the docstring**,
the descriptions shows only the first line.

![gif001](https://user-images.githubusercontent.com/57204023/132867114-94d7cec8-9274-43be-9f77-8df0108fb51c.gif)

## Steps to reproduce:

Write a function like the one show above with multiline parameters descriptions in the docstring, and try to use it,
or try to use a function that already exists.

Here is the snippet, to help:

```
def foo(a:int,b:float,c:str):
    """
        Parameters
        ----------
            a (int): integer number.
            b (float): description that takes
                more than 1 line in docstr
            c (str): word.
    """
    pass
```

**Comment by jakebailey** [comment]
This loose parameter parsing for signature help is one of the older parts of the code, and predates our improved docstring formatting; this could stand to be improved, though I wonder how much it's actually working.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by diogo-rossi** [comment]
Just to give an example. The JavaScript support in VS Code does that very well.

See this example.

```JavaScript
/**
 * Takes 2 numbers and return the sum
 * @param {Number} a The first number with very long description in it
 *  taking more than one line in the description
 * @param {Number} b The seconde number.
 * @returns The sum of the two numbers
 */

function sumNumbers(a, b){
    return a + b
}
```

### Calling the function

![image](https://user-images.githubusercontent.com/57204023/182623869-cfd914ef-c20e-44e9-9028-903358c3e5f3.png)

### The parameters

![image](https://user-images.githubusercontent.com/57204023/182624078-9776c695-c023-4ddc-9bec-dd61d9a321e3.png)

**Comment by EdNutting** [comment]
My vote: Please fix this. Thanks!
